test/templates/api/projects/instance-delete/004-delete-instance-ok/expected-response.json

```diff
@@ -1 +1,8 @@
-{}
+{
+  "id": "template.delete/%s",
+  "type": "template.delete",
+  "url": "https://templates.keboola.local/v1/tasks/template.delete/%s",
+  "status": "processing",
+  "isFinished": false,
+  "createdAt": "%s"
+}
```

Comment:
You should also check the task result, in both test.

See example in the Jira.